src/frontend/src/utils/userNumber.test.ts

```diff
@@ -90,11 +91,11 @@ function testLocalStorage(
 ) {
   test(name, () => {
     localStorage.clear();
-    if (opts?.before !== undefined) {
+    if (nonNullish(opts) && nonNullish(opts?.before)) {
       setLocalStorage(opts.before);
```

Comment:
```suggestion
    const before = opts?.before;
    if (nonNullish(before)) {
      setLocalStorage(before);
```


what do you think about this? I think it makes more sense